libs/tracing/dune

```diff
@@ -0,0 +1,15 @@
+; used to be in pfff in commons/Common.ml
```

Comment:
remove this comment

Reply:
or say instead that this is mostly a copy of ppx_profiling but using ocaml-trace